mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -71,6 +71,7 @@ CMomentumStickybombLauncher::CMomentumStickybombLauncher()
     m_bEarlyPrimaryFire.Set(false);
 }
 
+#ifdef GAME_DLL
```

Comment:
While I do understand that fizzle doesn't do anything on client right now, it may in the future so I would have to disagree with the ifdef guarding of the destructor here.

Reply:
Added the checks around the contents of it instead of the whole destructor. Figured this was better than putting the checks around the `Fizzle()` call, to prevent the loop.